src/command_modules/azure-cli-eventgrid/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.9
++++++
+* Updated to use the 0.4.0 EventGrid Python SDK.
```

Comment:
Since there are no customer-facing changes, please just put "Minor fixes." as the release notes.